tensorboard/plugins/interactive_inference/witwidget/notebook/colab/wit.py

```diff
@@ -275,7 +275,7 @@ def infer(self):
           inferences=json.dumps(inferences)))
     except Exception as e:
       output.eval_js("""backendError({error})""".format(
-          error=json.dumps({'msg': str(e)})))
+          error=json.dumps({'msg': repr(e)})))
```

Comment:
Could you comment on why we changed from str to repr?

Reply:
repr(error) returns a string that includes the error type, like "KeyError: ...", whereas str(e) will just return the detailed text, without the context of what type of error occurred.